Detectors/ITSMFT/ITS/reconstruction/src/CAaux.cxx

```diff
@@ -43,7 +33,7 @@ Track::Track(float x, float a, float* p, float* c, int *cl) :
   }
 
 bool Track::Update(const Cluster *cl) {
-  float p[2] = {cl->y,cl->z};
+  array<float,2> p{cl->y,cl->z};
   if (!mT.Update(p,cl->cov)) return false;
```

Comment:
@mpuccio Hi Max, not 100% sure, but please check: 1st of all, the GetPredictedChi2 should be calculated using the mT track kinematics **before** the mT.Update. 2nd, you are assigning to track chi2 the last track-to-cluster chi2, usually we are incrementing track chi2 by track-to-cluster chi2. To me, these lines should look like
```
float dChi2 = mT.GetPredictedChi2(p,cl->cov);
if (!mT.Update(p,cl->cov)) return false;
else SetChi2(dChi2+GetChi2());
return true;
```

Cheers,
 Ruben

PS: just for my education: is there any advantage of `array<float,2> p{cl->y,cl->z};` vs `float p[2] = {cl->y,cl->z};` ?